scripts/test-ci-change-scope.ps1

```
@@ -0,0 +1,141 @@
+
+        Write-Fixture 'src/Sample.cs' @'
+namespace Fixture;
+
+internal static class Sample
+{
+    /// <summary>A better explanation.</summary>
+    internal static string Value => "// this is a string";
+}
+'@
+        $comments = Save-Fixture 'comments only'
+        Assert-Mode 'csharp-trivia-only' (Get-Scope $initial $comments) 'comment-only change'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Assert-Mode' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)